tooling/noir_js_backend_barretenberg/src/base64_decode.ts

```diff
@@ -0,0 +1,13 @@
+// Since this is a simple function, we can use feature detection to
+// see if we are in the nodeJs environment or the browser environment.
+export function base64Decode(input: string): Uint8Array {
```

Comment:
We want to move this into noir-lang/types along with serialize, so that both noir-js and this package can access it